test/benchmark/test-benchmark-url.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
+
+// TODO(@anonrig): Remove this check when Ada removes ICU requirement.
+if (!common.hasIntl) {
+  // A handful of the benchmarks fail when ICU is not included.
+  // ICU is responsible for ignoring certain inputs from the hostname
```

Comment:
This file shouldn’t be reverted as well.

Reply:
I think it should be, `url.parse` depends on ICU if this PR lands.